homeassistant/helpers/event.py

```diff
@@ -349,41 +353,323 @@ def remove_listener() -> None:
     return remove_listener
 
 
+_boolean_coerce = vol.Any(cv.boolean, lambda val: False)
```

Comment:
Can't we replace this with `is True` ?

Reply:
And a comment to clarify why it's like that.

Reply:
That one is a bit weird.   If I can't get rid of it, I'll add a comment block

Reply:
I renamed it and moved it to `template.py`